closure/goog/events/browserevent.js

```diff
@@ -223,8 +223,16 @@ goog.events.BrowserEvent.IEButtonMap = [
 goog.events.BrowserEvent.prototype.init = function(e, opt_currentTarget) {
   var type = this.type = e.type;
 
+  /**
+   * On touch devices use the first "changed touch" as the relevant touch.
+   * @type {Touch}
+   */
+  var relevantTouch = e.changedTouches ? e.changedTouches[0] : null;
+
   // TODO(nicksantos): Change this.target to type EventTarget.
-  this.target = /** @type {Node} */ (e.target) || e.srcElement;
+  this.target = goog.isNull(relevantTouch) ?
+      /** @type {Node} */ (e.target) || e.srcElement :
+      /** @type {Node} */ (relevantTouch.target);
```

Comment:
> Do you have any clue how this could differ from e.target?

FYI, that particular change (`this.target`) came from an old patch: https://code.google.com/p/closure-library/issues/detail?id=382.

So here we're talking about initializing a `goog.events.BrowserEvent` instance from a touch event, i.e. a browser event resulting from a touch action on the touch surface.

And we want to provide the most relevant values for `this.client*`, `this.screen*` and `this.target`, `this` being the `goog.events.BrowserEvent` instance.

This PR suggests relying on `e.changedTouches[0]` for `this.client*` and `this.screen*`, as discussed already.

Now we're discussing `this.target`, trying to figure out whether it should be set to `e.target` or `e.changedTouches[0].target`.

So `e.target` refers to the `Element` the browser event was dispatched to (see [this mdn page](https://developer.mozilla.org/en-US/docs/Web/API/Event/Comparison_of_Event_Targets) for a good reference). `e.changedTouches[0].target` is the `Element` associated with the touch itself, which may be totally unrelated to the `e.target`. `this.target` should refer to the event's target, so I think it makes more sense to set always set it to `e.target`, even when `e` is a touch event.

If you agree, I can just remove that particular change from my branch.